thinc/tests/layers/test_layers_api.py

```diff
@@ -129,6 +130,8 @@ def assert_data_match(Y, out_data):
     ("MultiSoftmax.v1", {"nOs": (1, 3)}, array2d, array2d),
     # ("CauchySimilarity.v1", {}, (array2d, array2d), array1d),
     ("ParametricAttention.v1", {}, ragged, ragged),
+    ("ParametricAttention.v2", {}, ragged, ragged),
+    ("ParametricAttention.v2", {"key_transform": {"@layers": "Gelu.v1"}}, ragged, ragged),
```

Comment:
We should also have a test to check that the `key_transform` layer is actually inserted in the model, as opposed to when it's `None`.

Reply:
Added a test that checks if the transform is added to the model.